src/sec_edgar_bulker/core/downloader.py

```
@@ -0,0 +1,500 @@
+            submission: Submission data
+            pbar: Progress bar to update
+        """
+        try:
+            documents = await self._process_submission(submission)
+            if documents:
+                # Save results
+                results_file = self.results_dir / f"results_{submission['master_file'].stem}.jsonl"
+                async with aiofiles.open(results_file, "a") as f:
+                    for doc in documents:
+                        await f.write(json.dumps(doc) + "\n")
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)